lib/gameComponents/deck.dart

```diff
@@ -1 +1,30 @@
-class Deck {}
+import 'card.dart';
+import 'hand.dart';
+
+class Deck {
+  List<Hand> hands = [];
+  List<Card> cards = [];
+
+  Deck() {
```

Comment:
It might be worth having a look at multiple initialisers for this `Deck` class - as when it comes to testing (this class would be a great candidate for a test to ensure cards are actually shuffled), not having control of the deck could be a pain - e.g. if you want to test a function that doesn't require the deck to be shuffled, like taking the top card off the deck, you'd want to be able to control what card is on the top of the deck in your tests.

https://dart.dev/guides/language/language-tour#named-constructors

E.g. you could have something like this:
```dart
class Deck {
  //.....

   // This allows us to create a deck from scratch.
   // Why would this be important? It would allow for easier testing down the line, where you may only want to add
   // 4 cards to a deck, rather than having to deal with 52 when writing `expect`
  const Deck(this.cards);

  // Or during production running code, we could call this to get a shuffled deck of 52 cards
  factory Deck.createShuffled() {
    // Create a shuffled deck of 52 cards here
  }
}
```